<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskEdit</class>
 <widget class="QWidget" name="TaskEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>301</width>
    <height>154</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QPushButton" name="addButton">
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="cancelButton">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <layout class="QGridLayout" name="optionsLayout">
     <item row="2" column="4">
      <widget class="QComboBox" name="styleCombo">
       <property name="toolTip">
        <string>Style to hilight special tasks</string>
       </property>
       <item>
        <property name="text">
         <string>Regular</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Bold</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Italic</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="descLabel">
       <property name="text">
        <string>Description</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="multiLabel">
       <property name="text">
        <string>Max multiplier</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="typeCombo">
       <property name="toolTip">
        <string>Task type (either checkbox or a multiplier task)</string>
       </property>
       <property name="maxCount">
        <number>10</number>
       </property>
       <item>
        <property name="text">
         <string>Check</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Multi</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="pointsLabel">
       <property name="text">
        <string>Points</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="typeLabel">
       <property name="text">
        <string>Type</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="4">
      <widget class="QLineEdit" name="descEdit"/>
     </item>
     <item row="1" column="4">
      <widget class="QSpinBox" name="multiInteger">
       <property name="toolTip">
        <string>Maximum point multiplier</string>
       </property>
       <property name="maximum">
        <number>999</number>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="4">
      <widget class="QLineEdit" name="nameEdit">
       <property name="placeholderText">
        <string>Type task name here</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="pointsInteger">
       <property name="toolTip">
        <string>Points for completion</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLabel" name="styleLabel">
       <property name="text">
        <string>Style</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2" rowspan="2">
      <spacer name="optionsSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
